presentation-abstraction-control/src/main/java/com/iluwatar/pac/App.java

```diff
@@ -0,0 +1,28 @@
+package com.iluwatar.pac;
+
+import lombok.extern.slf4j.Slf4j;
+
```

Comment:
Add here Javadoc explaining the pattern briefly and how this example implements it